build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -39,7 +47,7 @@
   </Target>
   <Target Name="_BuildRuntimes"
       Inputs="@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\.stamp')"
-      Outputs="@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\mono\mini\.libs\%(OutputRuntime)');@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\mono\profiler\.libs\%(OutputProfiler)');@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\support\.libs\%(OutputMonoPosixHelper)')">
+      Outputs="@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\mono\mini\.libs\%(OutputRuntime)');@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\mono\profiler\.libs\%(OutputProfiler)');@(_MonoRuntime->'$(IntermediateOutputPath)%(Identity)\support\.libs\%(OutputMonoPosixHelper)');@(_BclProfileItems)">
```

Comment:
I'm not sure that `@(_BclProfileItems)` should be an output of `_BuildRuntimes`. On the one hand, yes, `make` in the `host-$(HostOS)` directory -- done as part of `_BuildRuntimes` -- will create them.

On the other hand, if there's just a "host" failure, e.g. building class libs, and all the native libraries were properly built, having these outputs as part of `_BuildRuntimes` means that they need to be rebuilt. Is that a problem? I'm not sure.